lib/tasks/conference.js

```diff
@@ -118,7 +118,10 @@ class Conference extends Task {
     this.emitter.emit('kill');
     await this._doFinalMemberCheck(cs);
     if (this.ep && this.ep.connected) {
-      this.ep.conn.removeAllListeners('esl::event::CUSTOM::*');
+      // IMPORTANT!!!! Remove CUSTOM event will make the ep is longer usable for any orther activity
+      // all freeswitch modules are firing CUSTOM events
+      // There is issue after caller is kicked from conferrence, he cannot use gather/transcribe
+      // this.ep.conn.removeAllListeners('esl::event::CUSTOM::*');
```

Comment:
perhaps we need to study this further, and figure out how to remove only event listeners for conference events, because I am pretty sure this code was in there for a reason (ie removing it may surface old bugs)

Reply:
You are right.

I have created new PR in fsmrf: https://github.com/drachtio/drachtio-fsmrf/pull/99

that allow us to reset the esl CUSTOM event instead of completely remove it.

Reply:
fixed, tested new change on jambonz.me and it's working